charts/vault/Makefile

```diff
@@ -0,0 +1,32 @@
+SHELL := /bin/bash # Use bash syntax
+ROOT_DIR:=../..
+include $(ROOT_DIR)/Makefile.env
+include $(ROOT_DIR)/hack/make-rules/docker.mk
+
+KUBE_NAMESPACE ?= m4d-system
+WITHOUT_OPENSHIFT ?= true
+
+.PHONY: install
+install:
+	kubectl create namespace $(KUBE_NAMESPACE) 2>/dev/null || true
+	helm repo add hashicorp https://helm.releases.hashicorp.com
+        ifeq ($(WITHOUT_OPENSHIFT), true)
+	$(info installing Vault without openshift)
+	helm install vault hashicorp/vault --values values/plugin-secrets-kubernetes-reader.yaml
+        else
+	helm install vault hashicorp/vault --values values/plugin-secrets-kubernetes-reader.yaml --values values/openshift.yaml
```

Comment:
Not setting dev mode? I don't think that the intention of this makefile is for production deployment

Reply:
its in the values.yaml file. I would set it outside but there are another flags that are related to dev mode: -dev-plugin-dir and VAULT_DEV_LISTEN_ADDRESS.

Reply:
VAULT_DEV_LISTEN_ADDRESS is a workaround for a bug in 0.9.0.
No longer needed in 0.9.1. See [changelog](https://github.com/hashicorp/vault-helm/blob/master/CHANGELOG.md#091-february-2nd-2021).

Reply:
Did you consider using proper plugin dir configuration (not for dev)?

Reply:
I will try to use that... tnx

Reply:
@roee88 I checked that and in order to configure the plugin that way, Vault server should not be in dev mode as it needs the plugin_directory to be in a configuration file, please see:  https://www.vaultproject.io/docs/configuration and 
https://www.vaultproject.io/docs/configuration#plugin_directory
Should I switch to non dev mode? Thanks